jquery-3.5.1.min.js

```diff
@@ -0,0 +1,2 @@
+/*! jQuery v3.5.1 | (c) JS Foundation and other contributors | jquery.org/license */
```

Comment:
Can we delete this file and just get jQuery from a CDN?

Reply:
Hmm - the index.html file already does get it remote, so this must just be from earlier testing.